core/templates/pages/library-page/library-page.component.ts

```diff
@@ -509,10 +509,7 @@ export class LibraryPageComponent {
   }
 
   moveClassroomCarouselToNextSlide(): void {
-    if (
-      this.currentCardIndex <
-      this.classroomSummaries.length - this.cardsToShow
-    ) {
+    if (this.currentCardIndex < this.classroomSummaries.length - 1) {
```

Comment:
Please help me to understand why we subtracted 1 here.

Ditto below for 2 and 3. Consider adding a comment in the code for understanding.

Reply:
We show 3 classroom cards and slide by 1. The first part is where we find the number of windows of size 3 and the other conditions are to check are we at the last window or not.

I have removed the hard-coded values and used cardsToShow state.